test/any.spec.js

```diff
@@ -0,0 +1,34 @@
+require('../aee.js')
+const expect = require('chai').expect
+const sinon = require('sinon')
+const data = require('./data')
+
+describe('#any', () => {
+  it('checks if any() is true when spec is a function and a match was found', () => {
+    function spec (element, index) {
+      return element.age === 24
+    }
+    var specSpy = sinon.spy(spec)
+    expect(data.people.any(specSpy)).to.equal(true)
+    expect(specSpy.callCount).to.equal(6)
```

Comment:
You could try to be less egocentric and find someone else than yourself on the list of persons, that will allow you to test your code stops calling `spec` when it finds the match.